lib/logging/monitor.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Logging
+  class Monitor < ::ZeroSilentFailures::Monitor
```

Comment:
probably should not inherit the ZSF monitor.  try and keep this a module/include/mixin with standalone functions
i think this will cause confusion too with 'circular inheritance'